guide/getting-started/hardware.md

```diff
@@ -54,15 +54,17 @@ General purpose hardware, like personal computers and smartphones, work great fo
    height = 400
 %}
 
-Many users back up their wallet using metal plates that have their recovery phrase etched into them. This protects the recovery phrase from physical destruction from fire and water. [Seedplate](https://seedplate.com/) is an example of such a product.
+Many users back up their wallet using stainless steel metal plates that have their recovery phrase etched into them. This protects the recovery phrase from [various forms](https://blog.lopp.net/metal-bitcoin-seed-storage-stress-test-iv/) of physical destruction. [Seedplate](https://seedplate.com/) is an example of such a product.
 
-A simpler option is writing your recovery phrase on a piece of paper. Paper can be easily destroyed compared to steel, making this option less secure.
+A simpler option is writing a recovery phrase on a piece of paper. Paper can be easily destroyed compared to steel, making this option less secure.
 
 Some applications, such as [Muun](https://blog.muun.com/why-not-just-a-mnemonic/) have users print out encrypted PDF backups that contain all the required recovery data, not just the recovery phrase.
 
 See [bitcoin backups]({{ '/guide/private-key-management/backups/' | relative_url }}) for more details.
 
-## Hardware wallets
+## External signers
```

Comment:
```suggestion
## Hardware wallets
```
I noticed it says "also known as external signers" immediately below this, so assuming we should change the title back to hardware wallets.

Reply:
I thought I did change it back, my bad thanks for that !